### PR TITLE
Values from interface works

### PR DESCRIPTION
**Pull request by shakib19911122**

### CODE REVIEW

**Review by bdcoelho — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -64,7 +64,7 @@ <h3>Choose your Criteria</h3>
                       <option value="1">1</option>
                       <option value="2">2</option>
                       <option value="3">3</option>
-                      <option value="3">4+</option>
+                      <option value="4">4+</option>
```
Comment (shakib19911122) [inline index.html:67]: update value

File: index.html [hunk 2]
```diff
@@ -75,19 +75,19 @@ <h3>Choose your Criteria</h3>
...
```
Comment (shakib19911122) [inline index.html:78]: update value

File: index.html [hunk 3]
```diff
@@ -75,19 +75,19 @@ <h3>Choose your Criteria</h3>
...
             
                   <div class="input-field col s12" id="propertyType">
                     <select single>
                       <option value="" disabled selected>Choose your option</option>
-                      <option value="1">House</option>
-                      <option value="2">Apartment</option>
-                      <option value="3">Townhouse</option>
-                      <option value="3">Land</option>
-                      <option value="3">Retirement</option>
+                      <option value="House">House</option>
```
Comment (shakib19911122) [inline index.html:86]: update value

File: index.html [hunk 4]
```diff
@@ -75,19 +75,19 @@ <h3>Choose your Criteria</h3>
...
+                      <option value="Apartment">Apartment</option>
+                      <option value="Townhouse">Townhouse</option>
```
Comment (shakib19911122) [inline index.html:88]: update value

File: index.html [hunk 5]
```diff
@@ -75,19 +75,19 @@ <h3>Choose your Criteria</h3>
...
+                      <option value="Land">Land</option>
```
Comment (shakib19911122) [inline index.html:89]: update value

**Pull request merged**